source/PlayerInfo.cpp

```diff
@@ -1728,6 +1735,49 @@ bool PlayerInfo::HasTravelPlan() const
 
 
 
+// Verify the player's travel plan and destination are still valid, in case the flagship was changed and/or
+// any needed outfits were removed.
+bool PlayerInfo::HasInvalidTravelPlan() const
+{
+	if(travelDestination && !travelDestination->IsAccessible(flagship.get()))
+		return true;
+	if(!travelPlan.empty())
```

Comment:
Since this never gets called in performance-critical code, it'd make more sense to just avoid duplicating all the code below by instead just checking:

```C++
if(!travelPlan.empty() && !DistanceMap(*this).HasRoute(travelPlan.front())
    return true;
```

The only downside would be that it would then pop up the warning message if the plan was invalidated due to systems becoming "unexplored" (e.g. the Pug invasion). But, I wonder whether showing that message is really useful or necessary, anyway.